R/oauth-flow-auth-code.R

```diff
@@ -110,12 +116,16 @@ req_oauth_auth_code <- function(req, client,
 #' @param auth_params List containing additional parameters passed to `oauth_flow_auth_code_url()`
 #' @param token_params List containing additional parameters passed to the
 #'   `token_url`.
-#' @param host_name Host name used to generate `redirect_uri`
-#' @param host_ip IP address web server will be bound to.
-#' @param port Port to bind web server to. By default, this uses a random port.
-#'   You may need to set it to a fixed port if the API requires that the
+#' @param host_name Deprecated. Use `redirect_url` instead.
+#' @param host_ip IP address for the temporary webserver used to capture the
+#'   authorization code.
+#' @param port Port to bind the temporary webserver to. Used only when
+#'   `redirect_url` is `"http(s)://localhost"`. By default, this uses a random
+#'   port. You may need to set it to a fixed port if the API requires that the
 #'   `redirect_uri` specified in the client exactly matches the `redirect_uri`
 #'   generated by this function.
+#' @param redirect_url URL to redirect back to after authorization is complete.
```

Comment:
@hadley There is an existing `redirect_uri` argument on this help page (for `oauth_flow_auth_code_url()`) -- should I aim for consistency and use that naming convention here as well?

Reply:
Yeah, we should make them consistent.

Reply:
Done.